packages/static-website/src/update_policy_handler/index.ts

```diff
@@ -0,0 +1,114 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+/* eslint-disable import/no-extraneous-dependencies */
+import {
+  GetKeyPolicyCommand,
+  KMSClient,
+  PutKeyPolicyCommand,
+} from "@aws-sdk/client-kms";
+import {
+  GetBucketPolicyCommand,
+  PutBucketPolicyCommand,
+  S3Client,
+} from "@aws-sdk/client-s3";
+import {
+  CdkCustomResourceEvent,
+  CdkCustomResourceResponse,
+  Context,
+} from "aws-lambda";
+
+const s3Client = new S3Client({
+  region: process.env.REGION,
+});
+const kmsClient = new KMSClient({
+  region: process.env.REGION,
+});
+
+export const handler = async (
+  event: CdkCustomResourceEvent,
+  ctx: Context
+): Promise<CdkCustomResourceResponse> => {
+  const response: CdkCustomResourceResponse = {
+    StackId: event.StackId,
+    RequestId: event.RequestId,
+    LogicalResourceId: event.LogicalResourceId,
+    PhysicalResourceId: ctx.logGroupName,
+  };
+
+  switch (event.RequestType) {
+    case "Create":
```

Comment:
Think it's worth adding **Update** to ensure these policies are present in the event a user manually removes them?

Reply:
Yeah probably useful! If doing update you might need to add IDs to the policy statements you add and replace if they exist to avoid continually growing the bucket/key policies :)